### ISSUE TITLE
Cannot access Plug-n-Play Devices

### ISSUE THREAD

**Issue by Programming-Lynx** [tfeldmann/organize#68]
Hi everyone,
Just a Bug I wanted to report, you cannot organize any Plug-n-Play devices.I believe this would also effect any secondary hard drives.

I have been attempting to use this tool to help organize my USB stick(s) and have discovered that the system cannot actually identify the folder the USB stick is:
(/media/[USERNAME]/[USB NAME]/filesAndFolders)

Classification:
Bug / Feature Request Combo

tl;dr : You cant organize USB sticks

**Comment by tfeldmann** [comment]
Hi, can you elaborate on this one? Can you show your config file and does any error message appear?

**Comment by Programming-Lynx** [comment]
I have included both a screenshot of the error and the configuration file in the attached zip.
[Zip](https://send.firefox.com/download/9ed5b9cbba98ba03/#41ppEcxe9MnVGMDCxmkIWA)

The error is known as :
Path not found: [TARGET LOCATION]
It may be in part due to the spaces used in the name of the USB or username.
I hope I can find a Fix.
Thank You

**Comment by tfeldmann** [comment]
Your config:
```yaml
- folders:
      - //media/kai/KAI\'S\ USB/Family\ Media/sorter/
```

I guess the problem is the two forward slashes at the start of the folder name. Could you try this one?

```yaml
- folders:
      - /media/kai/KAI\'S\ USB/Family\ Media/sorter/
```

And the same one without escaping:
```yaml
- folders:
      - "/media/kai/KAI'S USB/Family Media/sorter/"
```

**Comment by Programming-Lynx** [comment]
I attempted your solutions.
The first (remove the // into a / ) failed as before.
I had attempted removing the escape characters before but I have never placed them into quotation marks. That has solved the problem.

Thank you for your help.
Lynx
